src/main/java/com/xceptance/xlt/mastercontroller/MasterControllerConfiguration.java

```diff
@@ -272,9 +281,13 @@ else if (!(tempDirectory.isDirectory() && tempDirectory.canWrite()))
         // user name/password
         userName = XltConstants.USER_NAME;
         password = getStringProperty(PROP_PASSWORD, null);
-        
+
         // do we want to keep the timer files compressed for efficency
         compressedTimerFiles = getBooleanProperty(PROP_COMPRESSED_TIMER_FILES, true);
+
+        // download options
+        downloadChunkSize = getLongProperty(PROP_DOWNLOAD_CHUNK_SIZE, 100_000_000L);
+        downloadAttempts = getIntProperty(PROP_DOWNLOAD_ATTEMPTS, 2);
```

Comment:
Do we need to ensure some lower boundaries for these settings, e.g. download attempts at least 1 and download chunk size at least 1024?

Reply:
Done.